### DISCUSSION TITLE
Multiple TCP connections are in TIME_WAIT state

### DISCUSSION THREAD

**Discussion by zsharon75** [rabbitmq/rabbitmq-server#10386] [Other]
### Describe the bug

After working with RabbitMQ on load system - it was noticed that Management Connections are kept open in TCP level - 
Using NetStat we observed >6600 connections (which most likely can be considered zombie) on RabbitMQ side - where on client Application side there were only ~150 connections

Client Application Side (NETSTAT)
![image](https://github.com/rabbitmq/rabbitmq-server/assets/34444917/20aaa49b-1990-4bbb-885d-de717d57825f)

RabbitMQ Network side (NETSTAT)
![image](https://github.com/rabbitmq/rabbitmq-server/assets/34444917/97d11fc2-919b-4aad-8fc5-b5e40fe13c87)

### Reproduction steps

1. Using RabbitMQ 3.11.4 on Windows Server 2019
2. Using EasyNetQ.dll (.NET) - 6.3.1 - on Windows Server 2019 (part of client application)
3. Running application overnight.

### Expected behavior

RabbitMQ should properly close TCP connection on server side.

**Comment by michaelklishin** [comment]
RabbitMQ does not implement TCP. 

We do not have any evidence provided that would demonstrate this is due to something in RabbitMQ and not a [well known phenomenon of how TCP implements connection termination](https://vincent.bernat.ch/en/blog/2014-tcp-time-wait-state-linux), with an explicit and non-configurable 60s period in the `TIME_WAIT` state, that exists for a reason.

**Comment by lukebakken** [comment]
@zsharon75 EasyNetQ.dll is probably making HTTP API requests and not closing the connections gracefully.

It would be great if you could provide code we can compile and run to see the same behavior.
